selfdrive/test/process_replay/migration.py

```diff
@@ -2,16 +2,58 @@
 
 from cereal import messaging
 from openpilot.selfdrive.test.process_replay.vision_meta import meta_from_encode_index
+from openpilot.selfdrive.car.toyota.values import EPS_SCALE
+from panda import Panda
 
 
-def migrate_all(lr, old_logtime=False, camera_states=False):
+def migrate_all(lr, old_logtime=False, panda_states=False, camera_states=False):
   msgs = migrate_sensorEvents(lr, old_logtime)
   msgs = migrate_carParams(msgs, old_logtime)
+  if panda_states:
+    msgs = migrate_pandaStates(msgs, old_logtime)
   if camera_states:
     msgs = migrate_cameraStates(msgs)
 
   return msgs
 
+def migrate_pandaStates(lr, old_logtime=False):
+  all_msgs = []
+  # TODO: safety param migration should be handled automatically
+  safety_param_migration = {
+    "TOYOTA PRIUS 2017": EPS_SCALE["TOYOTA PRIUS 2017"] | Panda.FLAG_TOYOTA_STOCK_LONGITUDINAL,
```

Comment:
realed that if you want to regen any other route that's a Prius this won't work. I think the real solution is taking the online fingerprinted carParams and updating the param pandaStates.

Reply:
hmm, process replay already runs fingerprinting to get CP, I could probably use that